validator/sawtooth_validator/journal/genesis.py

```diff
@@ -212,11 +217,17 @@ def _get_block_publisher(self, state_hash):
         """
         state_view = self._state_view_factory.create_view(state_hash)
         try:
+            class BatchPublisher(object):
+                def send(self, transactions):
+                    raise InvalidGenesisConsensusError(
```

Comment:
The SignupInfo for the genesis controller will be included in the genesis node. It has to otherwise no  nodes can claim blocks. The c test (min age of sign up info) won't apply until there are at least C blocks are in the chain.